docsets/Spacemacs/docset.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Spacemacs",
+    "version": "0.200",
+    "archive": "spaceemacs.tgz",
```

Comment:
Your archive is called `Spacemacs.tgz` instead of `spaceemacs.tgz`.

Reply:
sorry typo, fixing